tron-transactions/templates/substreams.yaml.gotmpl

```diff
@@ -0,0 +1,34 @@
+specVersion: v0.1.0
+package:
+  name: {{ .ModuleName }}
+  version: v0.1.0
+
+imports:
+  tron: https://spkg.io/v1/packages/tron_foundational/v0.1.2
+
+protobuf:
+  excludePaths:
+    - sf/substreams
+    - google
+
+binaries:
+  default:
+    type: wasm/rust-v1
+    file: ./target/wasm32-unknown-unknown/release/substreams.wasm
+
+modules:
+ - name: map_my_filtered_transactions
+   use: stellar:filtered_transactions
+
+ - name: map_my_data
+   kind: map
+   initialBlock: 0
+   inputs:
+   - map: map_my_filtered_transactions
```

Comment:
Name is wrong, `stellar` here in the `use`.

Also, why do we `use` to our own modules, is it have `params` work out of the box here?